scanner/src/main/kotlin/Scanner.kt

```diff
@@ -63,7 +63,7 @@ abstract class Scanner {
      *
      * @throws ScanException In case the package could not be scanned.
      */
-    fun scan(pkg: Package, outputDirectory: File): ScannerResult {
+    fun scan(pkg: Package, outputDirectory: File, downloadDirectory: File?): ScannerResult {
```

Comment:
How about assigning `null` by default to `downloadDirectory`?

Reply:
Done.